recipes/pcl/all/conanfile.py

```diff
@@ -0,0 +1,252 @@
+        # https://github.com/PointCloudLibrary/pcl/blob/pcl-1.13.1/common/include/pcl/point_struct_traits.h#L40-L46
+        self.requires("boost/1.82.0", transitive_headers=True)
+        self.requires("eigen/3.4.0", transitive_headers=True)
+        self.requires("flann/1.9.2")
+        if self.options.with_png:
+            self.requires("libpng/1.6.40")
+        if self.options.with_qhull:
+            self.requires("qhull/8.0.1")
+        if self.options.with_apps:
+            self.requires("qt/6.5.1")
+        if self.options.with_libusb:
+            self.requires("libusb/1.0.26")
+        if self.options.with_pcap:
+            self.requires("libpcap/1.10.4")
+        if self.options.with_opengl:
+            self.requires("opengl/system")
+            self.requires("glew/2.2.0")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["PCL_SHARED_LIBS"] = self.options.shared
+        tc.cache_variables["WITH_OPENMP"] = self.options.with_openmp
+        tc.cache_variables["WITH_LIBUSB"] = self.options.with_libusb
+        tc.cache_variables["WITH_PNG"] = self.options.with_png
+        tc.cache_variables["WITH_QHULL"] = self.options.with_qhull
+        tc.cache_variables["WITH_VTK"] = self.options.with_vtk
+        tc.cache_variables["WITH_CUDA"] = self.options.with_cuda
+        tc.cache_variables["WITH_PCAP"] = self.options.with_pcap
+        tc.cache_variables["WITH_OPENGL"] = self.options.with_opengl
+        tc.cache_variables["BUILD_tools"] = self.options.with_tools
+        tc.cache_variables["BUILD_apps"] = self.options.with_apps
+        tc.cache_variables["BUILD_examples"] = False
+        tc.cache_variables["WITH_QT"] = self.options.with_apps
+        tc.cache_variables["PCL_ONLY_CORE_POINT_TYPES"] = True
+        tc.generate()
+
+        tc = CMakeDeps(self)
+        tc.set_property("eigen", "cmake_file_name", "EIGEN")
+        tc.set_property("flann", "cmake_file_name", "FLANN")
+        tc.set_property("flann", "cmake_target_name", "FLANN::FLANN")
+        tc.set_property("glew", "cmake_file_name", "GLEW")
+        if self.options.with_qhull:
+            tc.set_property("qhull", "cmake_file_name", "QHULL")
+        tc.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+        for mod in ["Eigen", "FLANN", "GLEW", "PCap", "Qhull", "libusb"]:
```

Comment:
```suggestion
        for mod in ["Eigen", "FLANN", "GLEW", "Pcap", "Qhull", "libusb"]:
```
Oops, my bad. Sorry about that. A downside of testing recipes on Windows. :grin: